src/ui/README.md

```diff
@@ -1,11 +1,13 @@
 # Aqueduct UI
 
-The Aqueduct UI is implemented in Next.js. Once you've installed the Aqueduct
-package, you can start the UI by running `aqueduct start`.
+The Aqueduct UI is implemented in React and Typescript. Once you've installed the Aqueduct
```

Comment:
Let's make it clear that installing the Aqueduct package is done via `pip` and not `npm` or something else.

Reply:
Sounds good, adding that in now.